base/keycloak-deployment/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
       labels:
         app: keycloak
     spec:
+      imagePullSecrets: 
+        - name: pull- 
```

Comment:
is that really needed?

Reply:
Hoops, leftover from previous iteration. Will remove it